Makefile

```diff
@@ -6,16 +6,15 @@ all: images
 images: lifemonitor
 
 certs:
-	if [[ ! -d "certs" ]]; then \
+		rm -rf certs && \
```

Comment:
This results in regenerating the certificates every time we run a `start*` rule.  I don't think that's desirable.

Reply:
I made this change after getting to a point (due to a previous docker build failure, I think) where I had an empty `certs` dir, so the lifemonitor image was being built correctly (since the check here is on dir existence) but it was actually broken due to missing certificates. I also ran into problems earlier because I tried to run `make startdev` after building the lifemonitor image manually, since I hadn't noticed that its dockerfile requires a (full) `certs` dir now (so I also got an image with missing certificates). So I think we can avoid these changes but we should add some notes to the docs.

Reply:
https://github.com/crs4/life_monitor/pull/27

Reply:
Also opened issue #29 for this.  The build process should never generate a broken image.